packages/electrode-archetype-react-app/lib/logger.js

```diff
@@ -0,0 +1,23 @@
+const winston = require("winston");
```

Comment:
also, should use `devRequire` to load winston since that's a dep pulled in by the dev-archetype.